landoui/dockerflow.py

```diff
@@ -16,8 +19,15 @@ def heartbeat():
     and return a 200 iff those services and the app itself are
     performing normally. Return a 5XX if something goes wrong.
     """
-    # TODO check backing services and update tests
-    return '', 200
+    try:
+        response = requests.get(
+            current_app.config['LANDO_API_URL'] + '/__lbheartbeat__'
```

Comment:
@purelogiq if you'd like more detail about the reasons why I decided to make this change, I'm happy to talk about it on IRC or vidyo, of course. :smiley: